composer.json

```diff
@@ -35,6 +36,7 @@
         "symfony/event-dispatcher": "dev-master",
         "symfony/monolog-bridge": "^3.2",
         "symfony/psr-http-message-bridge": "^0.2.0",
+        "willdurand/negotiation": "^2.3",
```

Comment:
isn't this dependency brought over from `elife/content-negotiator`?

Reply:
Yes, but it's touched in https://github.com/elifesciences/search/blob/7c5b5e1f10bca4b0e91740cc17199ad735c8dced/src/Search/Api/SearchController.php#L17